packages/icons/src/icons/regular/hamburger-default.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_2" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 16 16" style="enable-background:new 0 0 16 16;" xml:space="preserve">
+	 viewBox="0 0 16 16" style="enable-background:new 0 0 16 16;" xml:space="preserve" width="16" height="16">
```

Comment:
I think probably the best solution is working with Bryan to define these either in the Illustrator source files, or somehow embedding them in the export process. But the icon sizes are not consistent (they're not always 24 or 16 pixels) so there's no global solution for setting these. It's possible to derive it programmatically by looking at the viewbox attribute, which does tend to be correct and always specified.

@nfiniteset thoughts on this?

Reply:
Setting these manually is a problem. When Bryan gives us updated icons we will not remember to set these attributes again. If these attributes are needed we need to file a story to capture doing it programmatically.

Reply:
I can back out the SVG changes but the icons don't render correctly then. Which maybe isn't a huge deal because they don't render correctly in the `development` branch either.

I tried experimenting with the `max-width` css property but it doesn't look like the svg element pays attention to it.